src/main/java/com/Studentlifr/GratitudeBoard/dao/UserDAO.java

```diff
@@ -0,0 +1,39 @@
+package com.Studentlifr.GratitudeBoard.dao;
+
+import com.Studentlifr.GratitudeBoard.enumerators.ROLES;
+import jakarta.persistence.*;
+import lombok.*;
+import org.hibernate.annotations.UuidGenerator;
+
+import java.util.UUID;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@Table(name = "users")
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserDAO {
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @UuidGenerator
+    @Column(name = "id", updatable = false, nullable = false)
+    private UUID id;
```

Comment:
have consistent naming strategy for primary keys

either use `id` everywhere or use `userId` `messageId` .. so on